src/types/PhoneCard.ts

```diff
@@ -0,0 +1,15 @@
+export interface PhoneCard {
```

Comment:
```suggestion
export interface Phone {
```
And why i think its important) because it's like the main type we'll have in this project and everyone will use it. So the interface is perfect :fire: . But the name of type and its directory should be clear for everyone, and basically it's just a phone)))